pages/api/routine_logs/[routineName]/summary.js

```diff
@@ -15,7 +22,7 @@ export default async function handler(req, res) {
         exceptions: exceptions,
         warnings: warnings,
         info: infos,
-        avgFps: 1
+        avg_fps: fps
```

Comment:
Consider using the following:

```suggestion
        avg_fps: fpsList[0] ? fpsList[0]["fps"] : "0"
```

It allows you to keep the async db call as part of the `Promise.all(...)`